packages/pf-codemods/README.md

```diff
@@ -324,3 +324,21 @@ Out:
 ```jsx
 <Tooltip     />
 ```
+
+### chart-resizeObserver-function [(#8533)](https://github.com/patternfly/patternfly-react/pull/8533)
```

Comment:
Nit: can this be placed alphabetically in the README

Reply:
Done